style/index.less

```diff
@@ -0,0 +1,18 @@
+.content {
```

Comment:
Note that `.content` is relevant to our demo page, not so much for external consumer of the library. Same for the margin below.

Reply:
Right, leaving the `.content` css in index.html then